### DISCUSSION TITLE
Subtitle subfolder

### DISCUSSION THREAD

**Discussion by ajpanton** [jellyfin/jellyfin#12466] [Q&A]
Within my main movie folder I have a subfolder named "subs", in order to not clutter the folder for the users who just browse the network share and load the movies manually using whatever media player they have.

Jellyfin can read some of the subtitles in there, but not all of them. There's no logic in it, like "movie.mkv" will load "movie.eng.srt" and "movie.fin.srt", but not "movie.swe.srt". Most subtitles will not load however, so I'm wondering if there's anything that can be done?

**Comment by TheMelmacian** [comment]
Can jellyfin read them when they are in the same folder as the movie?
What I do to prevent cluttering of the main movie folder is to pack all external subtiles in a single *.mks file.

**Comment by mircolosi** [comment]
I found this plugin that should work with subtitles placed in subfolders:
https://github.com/azam/jellyfin-plugin-localsubs
